docs/components/sliders.md

```diff
@@ -0,0 +1,400 @@
+| Show Value Indicator Shape | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `showValueIndicatorShape` parameter. |
+
+#### Tick mark attributes
+
+| &nbsp; | Attribute |
+|---|---|
+| Tick Mark Shape | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `tickMarkShape` parameter. |
+| Range Tick Mark Shape | Wrap your `RangeSlider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `rangeTickMarkShape` parameter. |
+| Active Tick Mark Color | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `activeTickMarkColor` parameter. |
+| Inactive Tick Mark Color | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `inactiveTickMarkColor` parameter. |
+| Disabled Active Tick Mark Color | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `mi` parameter. |
+| Thumb Selector | Wrap your `Slider` widget with a `SliderTheme` widget within the data parameter use the `SliderThemeData` widget. Within this widget set the `thumbSelector` parameter. |
+
+### Continuous sliders
+
+Continuous sliders allow users to make meaningful selections that don’t require
+a specific value.
+
+#### Continuous single point slider example
+
+API and source code:
+
+- [Class definition](https://api.flutter.dev/flutter/material/Slider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/slider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/sliders/continous_slider&gh_ref=develop)
+
+!["Continous Slider"](assets/sliders/continous_slider.png)
+
+```dart
+double _sliderValue = 20;
+
+
+Slider(
+    value: _sliderValue,
+    min: 0,
+    max: 100,
+    label: _sliderValue.round().toString(),
+    onChanged: (value) {
+        setState(() {
+            _sliderValue = value;
+        });
+    },
+),
+```
+
+API and source code:
+
+- [Class definition](https://api.flutter.dev/flutter/material/Slider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/range_slider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/sliders/continous_range_slider&gh_ref=develop)
```

Comment:
This should all be below the header beneath it

Reply:
Done